README.md

```diff
@@ -34,3 +34,7 @@ You can find more detailed documentation in the [`./docs` folder](https://github
 ## Contributing
 
 Although this project is foremost developed to fit the needs of [YOMA](yoma.africa) it eventually is an open-source project that hopefully can get picked up and used for other aries-hyperledger projects as well. We do welcome contributions from anyone. Feel free to engage in discussions, create a PR against a ticket or open an issue yourself following the GitHub guidelines.
+
+## Running tests
```

Comment:
Can we add a non-make way to run the python tests? Make is not a tool that is familiar to all developers (I never use the make command) so would be good to also highlight how to run it directly.

Reply:
sure